### ISSUE TITLE
CCDr Algorithm + estimate.dag in R Script, Error in weights

### ISSUE THREAD

**Issue by Christopher7622** [FenTechSolutions/CausalDiscoveryToolbox#158]
the details of this error:
![image](https://github.com/FenTechSolutions/CausalDiscoveryToolbox/assets/92440203/e7a53ccc-3970-47bc-a535-d6399b0df3ac)

I use traceback() function and get some information of estimate.dag():
![image](https://github.com/FenTechSolutions/CausalDiscoveryToolbox/assets/92440203/9c069c33-ee8d-456a-8cc4-32a97990485b)

my R version is 4.2.1 ,and I have installed sparsebn,sparsebnUtils and MASS ,I used to run example provided in rdrr.io website,the same error occurred.what is the problem?

**Comment by diviyank** [comment]
Hello, could you show, if possible, an extract of your data?

**Comment by diviyank** [comment]
Duplicate of #157 I think CCDr is broken, it will need some recoding

**Comment by Christopher7622** [comment]
> Hello, could you show, if possible, an extract of your data?

Of course,the dataset i am useing is the Sachs dataset provided in CausalDiscoveryToolBox.
[cyto_full_data.csv](https://github.com/FenTechSolutions/CausalDiscoveryToolbox/files/13703351/cyto_full_data.csv)
[cyto_full_target.csv](https://github.com/FenTechSolutions/CausalDiscoveryToolbox/files/13703352/cyto_full_target.csv)
